src/index.ml

```diff
@@ -181,7 +181,7 @@ module Make (K : Key) (V : Value) (IO : IO) = struct
       if readonly then None
       else Some (Bloomf.create ~error_rate:0.01 100_000_000)
     in
-    let log_mem = Tbl.create 1024 in
+    let log_mem = Tbl.create log_size in
```

Comment:
We used to have that initialization a few weeks ago, but reduced the initial size since there could be use cases where we add fewer entries than that and don't want to allocate extra memory.

Reply:
agreed, let's keep a small `log_mem` and let the OCaml runtime change it when needed.